### DISCUSSION TITLE
Leash Teleport

### DISCUSSION THREAD

**Discussion by hirata-estate** [PurpurMC/Purpur#538] [Feature Requests]
Feature that lets leashed mobs teleport with you, and a true/false setting if the leashes should break after teleporting.
There's already a plugin for this but it'll be great to just have it as a setting. Thank you!

**Comment by CheckForUpdates** [comment]
Not sure if it's a plugin feature or just not working correctly, but this seems to be happening by default for us. I can't figure out how to prevent it and I can't find an option in any of the configs. Maybe I'm overlooking something but it's extremely annoying.
